src/ui/public/notify/Notifier.js

```diff
@@ -93,6 +109,10 @@ define(function (require) {
     return err.stack;
   }
 
+  function infoAsObject(version, buildNum) {
```

Comment:
Is there a reason you have this defined as a helper, instead of just using `{ version, buildNum }` below?